package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "reality-capture-main",
+  "version": "1.0.0",
+  "description": "This project contains iTwin API SDK for ContextCapture and RealityDataAnalysis as well as reality data API utils, in typescript and python.",
+  "workspaces": [
```

Comment:
are workspaces supposed to be defined in here on in the pnpm-worksapce.yaml file

Reply:
removed field in package.json